providers/shared/references/DeploymentGroup/id.ftl

```diff
@@ -28,6 +28,12 @@
             "Values" : [ "", "account", "segment", "solution", "application" ],
             "Mandatory" : true
         },
+        {
+            "Names" : "District",
+            "Description" : "The id of the district for components included in the deployment group",
```

Comment:
Need to be consistent with our terminology. "The id of the district" is "segment" not the id of the solution within the district right? I think this is how we've used it up until now and how it is reflect in recent changes to the cli.

When we come to a solution within a district, that's when we talk about the id of the solution.

Reply:
Good point, currently deployment groups live outside of the solution as part of the reference data. District seemed the best fit here so that the groups can be used across multiple solutions but they are still tied to a given district. This ensures that you don't get overlap and can define the deployment easily

Reply:
all good.